bin/impl/fetch.sh

```diff
@@ -122,13 +116,20 @@ zookeeper)
   fetch_zookeeper
   ;;
 *)
-  echo "Usage: uno fetch <component>"
-  echo -e "\nPossible components:\n"
-  echo "    accumulo   Downloads Accumulo, Hadoop & ZooKeeper. Builds Accumulo if repo set in uno.conf"
-  echo "    fluo       Downloads Fluo, Accumulo, Hadoop & ZooKeeper. Builds Fluo or Accumulo if repo set in uno.conf"
-  echo "    hadoop     Downloads Hadoop"
-  echo "    zookeeper  Downloads ZooKeeper"
-  echo "Options:"
-  echo "    --no-deps  Dependencies will be fetched unless this option is specified. Only works for fluo & accumulo components."
+  cat <<EOF
+Usage: uno fetch <component>
+
+Possible components:
+
+    accumulo   Downloads Accumulo, Hadoop & ZooKeeper. Builds Accumulo if repo set in uno.conf
+    fluo       Downloads Fluo, Accumulo, Hadoop & ZooKeeper. Builds Fluo or Accumulo if repo set in uno.conf
+    hadoop     Downloads Hadoop
+    zookeeper  Downloads ZooKeeper
+
+Options:
+    --no-deps  Dependencies will be fetched unless this option is specified. Only works for fluo & accumulo components.
+EOF
   exit 1
 esac
+
+# fetch.sh
```

Comment:
What is the purpose of these comments?

Reply:
It's mostly style. They help me remember which file I'm editing, and help provide a non-empty final line for the script, so that I don't get warnings with `git add --patch`. But, they serve no real functional purpose.